roles/csi-driver/templates/app/demo.j2

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Can be removed?

Reply:
every ansible playbooks and config typically starts with ---.